Write introduction

daffidwilde opened the pull request:
An introduction consisting of:

- Background/brief review of population segmentation, queuing in healthcare, reverse-engineering model parameters in healthcare
- Condensed analysis of clustering found, and *maybe* some comparison to treatment-based segmentation
- Boilerplate what happens here and/or a figure showing the process of the paper

daffidwilde committed:
Take first stab at intro preamble.

Would this make a good abstract?

daffidwilde committed:
Save cluster analysis plots and table.

daffidwilde committed:
Add plots and table to text.

daffidwilde committed:
Begin writing literature review.

daffidwilde committed:
Continue lit review into segmentation analysis

daffidwilde committed:
Add section for dataset overview.

daffidwilde committed:
Update image formats for interventions.

daffidwilde committed:
Add to general analysis.

daffidwilde committed:
Update general age and WIMD images.

daffidwilde committed:
Continue writing and adding figures.

Expanded lit review; added figures for LTC proportions; add to the
summary table.

daffidwilde committed:
Add further images.

daffidwilde committed:
Finish lit review of segmentation; gather refs.

daffidwilde committed:
Move queuing refs to incomplete data subsec.

daffidwilde committed:
Work in minor review

daffidwilde committed:
Write missing data lit review.

daffidwilde committed:
Clean up notebooks and automate dataset details.

daffidwilde committed:
Start writing overview/clustering section.

daffidwilde committed:
Switch to spell-level statistics in table.

daffidwilde committed:
Add ICD category network notebook.

It might be interesting (for thesis) to have a look at the concurrent
ICD code categories as a network for each cluster. Maybe there is some
actual graph work to be done?

drvinceknight commented on sections/intro.tex:
```diff
@@ -1,6 +1,290 @@
+However, this process relies heavily on detailed data --- about both the system
+and the population within that system --- which may limit research where
+sophisticated data pipelines are not yet in place. This work demonstrates how
+this issue may be overcome using administrative, spell-level hospital data to
```
I'm not sure `spell-level` is a sufficiently common term? (I might be wrong)

daffidwilde replied:
Yes, I'd been thinking that. It's not super common in literature but that's because you'd normally have access to *all* the data. I'll skip over the details here and add more in the data overview section.

drvinceknight replied:
Sure. It might just need a single sentence saying what is meant by it and even repeating what you've said there (that normally all the data is available).

drvinceknight commented on sections/intro.tex:
```diff
@@ -1,6 +1,290 @@
+highest prevalence of the condition across all the Welsh health boards in an
+internal report by NHS Wales.
+
+% TODO No reference for graph sent by Kendal but is this alright?
```
Can you remind me what this is about?

Sounds like we'll need to at least check with Kendal but at best reference some original source.

daffidwilde replied:
It was to do with a figure that didn't come from a published article per se -- just an NHS report. I need to get in touch with Kendal anyway so I'll ask for his recommendation.

drvinceknight replied:
I'd wait on this, it'll come out in the wash when we have a draft for Kendal to look at.

daffidwilde replied:
Right you are

drvinceknight commented on sections/intro.tex:
```diff
@@ -1,6 +1,290 @@
+scenarios with insight provided by the model parameterisation and the
+clustering; Section~\ref{sec:conclusion} concludes the paper. Although the data
+is confidential and may not be published, the source code used in this paper is
+available online at~\url{https://github.com/daffidwilde/copd-paper}.
```
Can we generate a small synthetic data set as an example? We could archive this and include a notebook in the GitHub repo?

(Not suggesting we do something too clever to generate the synthetic data, just some random numbers will do to showcase the approach.)

daffidwilde replied:
Yeah, sure thing. I could do a kernel density fit on the real LOS data and sample for each cluster?

drvinceknight replied:
Something like that yup :+1:

daffidwilde replied:
That's done now and is archived at https://doi.org/10.5281/zenodo.3908167 👍

drvinceknight commented on sections/intro.tex:
```diff
@@ -1,6 +1,290 @@
+Given the subject matter of this work, the relevant literature spans much of
+operational research in healthcare and the focus of this review is on the
+principal topics of segmentation analysis, queuing models applied to hospital
+systems, and the handling of missing or incomplete data for such queues.
```
Nice.

drvinceknight commented on sections/intro.tex:
```diff
@@ -1,6 +1,290 @@
+number of parts cannot be found initially~\cite{Vuik2016a}. Although, supervised
+hierarchical segmentation methods such as classification and regression trees
+(as in~\cite{Harper2006}) have been used where an existing, well-defined label
+is of particular significance.
```
I thought the above section was really excellent Henry.

daffidwilde replied:
Thanks Vince

drvinceknight commented on sections/intro.tex:
```diff
@@ -1,6 +1,290 @@
+
+These are the principal queuing theory works by
+Erlang~\cite{Erlang1917,Erlang1920}. Deadlock is an aspect of applied queuing
+theory of interest in recent literature~\cite{Palmer2018}. The software used is
```
If there's a valid reason to reference `Palmer2018` then of course we include it but I don't immediately see how deadlock is related to this work. (As I'm an author on both this and that paper I wouldn't want to just self reference for the sake of it.)

daffidwilde replied:
Yeah, I've just got it as a placeholder for now 👍

drvinceknight commented on sections/intro.tex:
```diff
@@ -1,6 +1,290 @@
+These are the principal queuing theory works by
+Erlang~\cite{Erlang1917,Erlang1920}. Deadlock is an aspect of applied queuing
+theory of interest in recent literature~\cite{Palmer2018}. The software used is
+Ciw~\cite{Palmer2019}.
```
```suggestion
Ciw~\cite{Palmer2019}.  % TODO Write this section
```

I assume this paragraph isn't complete?

daffidwilde replied:
Correct. I'll add the `TODO` now.

drvinceknight commented on sections/intro.tex:
```diff
@@ -1,6 +1,290 @@
+
+\subsubsection{Handling incomplete queue data}
+
+It is often the case that in practical situations where suitable data is not
```
I think we might need a short sentence or two at the start of this `subsubsection` reminding the reader of the context here (that there is missing data).

daffidwilde replied:
Good idea

daffidwilde replied:
Fixed in e4bacb4

drvinceknight commented on sections/intro.tex:
```diff
@@ -1,6 +1,290 @@
+
+\subsection{Overview of the dataset and its clustering}\label{subsec:overview}
+
+The dataset used in this work was provided by the Cwm Taf Morgannwg UHB as part
+of an ongoing research project with the authors. The dataset contains a
```
```suggestion
The dataset used in this work was provided by the Cwm Taf Morgannwg UHB. The dataset contains a
```

drvinceknight commented on sections/intro.tex:
```diff
@@ -1,6 +1,290 @@
+
+The dataset used in this work was provided by the Cwm Taf Morgannwg UHB as part
+of an ongoing research project with the authors. The dataset contains a
+spell-level summary of \input{tex/npatients}patients presenting COPD from
```
Same comment about `spell-level` (it might be fine just making sure.)

daffidwilde replied:
Mmm.. I'll rework these sentences and introduce "spell-level" more formally

drvinceknight reviewed (changes requested):
Everything is looking good @daffidwilde (Figure 2 is fancy!)

daffidwilde committed:
Add TODO marker for queue lit review

Co-authored-by: Vince Knight <vincent.knight@gmail.com>

daffidwilde committed:
Remove "ongoing research" phrase

Co-authored-by: Vince Knight <vincent.knight@gmail.com>

daffidwilde committed:
Fix Larsen2017 citation

daffidwilde committed:
Merge branch 'write-intro' of github.com:daffidwilde/copd-paper into write-intro

daffidwilde committed:
Remove "spell-level" term.

daffidwilde committed:
Add more flesh to the "spell-level" sentences.

daffidwilde committed:
Remove duplicate spells from cluster images.

daffidwilde committed:
Add a refresher to incomplete data lit review.

daffidwilde committed:
Start queuing lit review.

daffidwilde committed:
Add references for COPD-deprivation links.

daffidwilde committed:
Add Sexton2016 reference to COPD-WIMD link.

daffidwilde committed:
Finish queuing part of lit review; first lit rev.

daffidwilde committed:
Fix percentages in spell table.

daffidwilde committed:
Re-run cluster analysis notebook

daffidwilde committed:
Add to cluster discussion.

daffidwilde committed:
Update cluster analysis images.

daffidwilde committed:
Update "accessed on" for misc citations.

daffidwilde committed:
Resize table and adjust discussion linebreaks.

daffidwilde committed:
Continue closing analysis of clusters.

daffidwilde committed:
Fix Rebuge2012 citation.

daffidwilde committed:
Finish draft of introduction.

daffidwilde committed:
Write short notebook for synthetic data creation.

daffidwilde committed:
Add citation for synthetic data; correct bibstyle.

daffidwilde committed:
Update kernel in data archive nb.

daffidwilde committed:
Add row for percentage of total costs.

As discussed in meeting with VK 2020-06-26, a row has been added to this
table summarising the percentage of total costs for each cluster.

daffidwilde committed:
Fix rounding in table.

daffidwilde committed:
Add a transition paragraph to end of intro.
